backend/Makefile

```diff
@@ -24,21 +26,49 @@ build-docker: test
 
 ## migrations
 install_goose:
-	go get -u github.com/pressly/goose/v3/cmd/goose
+	$(call install_goose)
 
 goose_up:
-	cd migrations && goose postgres $(PG_STRING) up
+ifeq (, $(shell which goose))
+	$(call install_goose)
+endif
+	cd ./scripts/ && ./goose_up.sh
 
 goose_down:
-	cd migrations && goose postgres $(PG_STRING) down
+ifeq (, $(shell which goose))
+	$(call install_goose)
+endif
+	cd ./scripts/ && ./goose_down.sh
+
+## start-api: starts the api and its dependencies in a docker container
+start-api:
+	docker-compose up
+
+## stop-api: stops the api and its dependencies in a docker container
+stop-api:
+	docker-compose down && \
+	rm -rf postgres-data
```

Comment:
Not quite sure... but do we have rights on accessing this file, deleting it etc... ?
My question is due to the comment of @DipandaAser with errors he is having right now 🤔

Reply:
As i was suspecting, am getting the error when testing...
![Screenshot from 2021-09-14 19-00-39](https://user-images.githubusercontent.com/22576758/133301710-a2447ab6-c27b-4c84-b069-caa4fcd586ed.png)